MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+module(
+    name = "bazel_skylib",
+    compatibility_level = 1,
+    version = "1.2.1",
+)
+
+register_toolchains(
+    "//toolchains/unittest:cmd_toolchain",
+    "//toolchains/unittest:bash_toolchain",
+)
```

Comment:
https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/bazel_skylib/1.2.1/MODULE.bazel instead uses 

```
module(
    name = "bazel_skylib",
    compatibility_level = 1,
    toolchains_to_register = [
        "@bazel_skylib//toolchains/unittest:cmd_toolchain",
        "@bazel_skylib//toolchains/unittest:bash_toolchain",
    ],
    version = "1.2.1",
)
```

AFAICT, both `toolchains_to_register` and `register_toolchains` allow bazel_skylib's own tests to pass. Which one is preferred?